src/stories/Avatar/Avatar.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Avatar } from "./Avatar";
+import { AvtarConatiner } from "./Avatar.style";
+
+export default {
+  title: "Avatr ",
+  component: Avatar,
+};
+
+export const CircularProfileAvatar = () => (
+  <AvtarConatiner>
+    <Avatar
+      url="https://i.pravatar.cc/40"
+      type="small"
+      alt="avatar-2"
```

Comment:
don't give alt tag like avatar-2 please make proper name @PRABgit01

Reply:
ok sir i ll give proper naming to it